RAG_CHATBOT_BACKEND_APIS/app/services/Modules/Chatbot/ChatBotService.py

```
@@ -75,107 +75,157 @@ def get_user_chatbots(user_id):
 
     @staticmethod
-    def chat_bot_folder_strucuture(user_data , chat_bot_name):
-        smedia_file, media_bot_ = create_directories(str(user_data.username), str(chat_bot_name))
-        copy_path = os.path.join(settings.COPY_ROOT, 'chat_bot')
-        try:
-            copy_directory_contents(copy_path, smedia_file)
-            return [True, smedia_file, media_bot_]
-        except Exception as e:
-            logger.error(f"❌ Error creating new chatbot directories: {e}")
-            return [False,'' , '']
+    def chat_bot_folder_strucuture(user_data, chat_bot_name, update_type, chat_bot_data={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)